docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
```

Comment:
remove this entire section line, I'm not sure this page actually needs any subsections